tests/test_speech_to_text.py

```diff
@@ -62,22 +108,3 @@ def test_speech_to_text():
 
     jobs = queue.receive_messages(MaxNumberOfMessages=1)
     assert len(jobs) == 0, "queue empty"
-
-
-def clean():
-    """
-    Ensure that the bucket and queues are empty.
-    """
-    todo = speech_to_text.get_todo_queue()
-    while messages := todo.receive_messages():
-        for m in messages:
-            m.delete()
-
-    done = speech_to_text.get_done_queue()
-    while messages := done.receive_messages():
-        for m in messages:
-            m.delete()
-
-    bucket = speech_to_text.get_bucket()
-    for obj in bucket.objects.all():
-        obj.delete()
```

Comment:
Since we aren't using AWS live anymore, and moto resources are temporary, this cleanup can go away.